ramls/inventory-hierarchy/inventory-items-and-holdings.json

```diff
@@ -0,0 +1,804 @@
+                    "description": "The name of the institution, the first-level location unit, this (shelf) location belongs to",
+                    "type": "string"
+                  }
+                },
+                "required": [
+                  "name",
+                  "campusName",
+                  "libraryName",
+                  "institutionName"
+                ]
+              },
+              "temporaryLocation": {
+                "type": "object",
+                "description": "Temporary location, shelving location, or holding which is a physical place where items are stored, or an Online location",
+                "properties": {
+                  "name": {
+                    "description": "Name of the (shelf) location",
+                    "type": "string"
+                  },
+                  "campusName": {
+                    "description": "The name of the campus, the second-level location unit, this (shelf) location belongs to",
+                    "type": "string"
+                  },
+                  "libraryName": {
+                    "description": "The name of the library, the third-level location unit, this (shelf) location belongs to",
+                    "type": "string"
+                  },
+                  "institutionName": {
+                    "description": "The name of the institution, the first-level location unit, this (shelf) location belongs to",
+                    "type": "string"
+                  }
+                },
+                "required": [
+                  "name",
+                  "campusName",
+                  "libraryName",
+                  "institutionName"
+                ]
+              }
+            },
+            "required": [
+              "permanentLocation",
+              "temporaryLocation"
+            ]
+          },
+          "callNumber": {
+            "description": "Call Number is an identifier assigned to a holding",
+            "type": "object",
+            "properties": {
+              "prefix": {
+                "description": "Prefix of the call number on the holding level",
+                "type": "string"
+              },
+              "suffix": {
+                "description": "Suffix of the call number on the holding level",
+                "type": "string"
+              },
+              "typeId": {
+                "description": "Unique ID for the type of call number on a holdings record",
+                "type": "string",
+                "$ref": "../uuid.json"
+              },
+              "typeName": {
+                "description": "Name of the call number type",
+                "type": "string"
+              },
+              "callNumber": {
+                "description": "Call Number identifier assigned to a holding",
+                "type": "string"
+              }
+            },
+            "required": [
+              "prefix",
+              "suffix",
```

Comment:
How are these required when a call number does not have to have either a prefix or a suffix?

Reply:
I checked and updated requirements for payload elements.